docs/production-deployment/cloud/nexus/operations.mdx

```diff
@@ -201,7 +201,10 @@ See [Nexus Secure Routing](/nexus/security#secure-routing) for additional detail
 ## Payload Encryption
 
 For payload encryption, the DataConverter works the same for a Nexus Operation as it does for other payloads sent between a Worker and Temporal Cloud.
-Currently there is support for a single DataConverter to be used for inbound inputs and outbound outputs, which means the encryption key often used in a custom DataConverter must be the same for both the caller and the handler.
-Please let us know if you need per-Endpoint or per-Service payload encryption.
+The caller and handler Workers must have compatible Data Converters as operation inputs and results are passed between the two.
```

Comment:
Can we avoid duplicating the same point three times?

Reply:
The encyclopedia page has the full info. We do refer these other sections (evaluate, deploy to prod) to the encyclopedia entry, but keep a summary in these other section which IMO is useful to make folks aware, without getting into the details -- which is what the encyclopedia entry is for.

For new users, would expect them to read the evaluate section or deploy to prod sections first, so having a little context/summary there is goodness IMO to build awareness.

Reply:
I would rather not have so much duplication if possible. It gets hard to maintain and when a user browses these docs pages, they'll get tired from seeing the same content appear everywhere.

Reply:
I do agree with the duplication since it's the exact same on the security page. Could this sentence just be rephrased?